test/telemetry_metrics_statsd_test.exs

```diff
@@ -292,7 +293,7 @@ defmodule TelemetryMetricsStatsdTest do
                TelemetryMetricsStatsd.udp_error(reporter, udp, :closed)
                # Can we do better here? We could use `call` instead of `cast` for reporting socket
```

Comment:
```suggestion
```

This comment is outdated now.

Reply:
i'm ok with removing. 
But I didn't on purpose, as the original question remains. 
Can call be used instead of cast? if we do so, we will block the process calling telemetry, which might not be desirable. In that case, cast is the right choice, and we can safely remove